ape_hardhat/providers.py

```diff
@@ -30,6 +31,18 @@
     },
   },
 };
+
+task("dump-accounts", "Dumps accounts as JSON for the current project", async (taskArgs, hre) => {
```

Comment:
Since the `dump-accounts` is not needed anymore, could we remove that and release just the `call_hardhat_task` functionality?

The `dump-accounts` is a good test though... We could add it to the tests and use it there?

Thoughts?  I think this is really cool for some reason lol.